Makefile

```
@@ -1,262 +1,2 @@
-
-man: $(foreach lang,$(LANGS),man-$(lang))
-	$(SPHINXBUILD) -b man $(ALLSPHINXOPTS) $(BUILDDIR)/man/en
-	@echo
-	@echo "Build finished. The manual pages are in $(BUILDDIR)/man/en"
-man-%: locale-%
-	$(SPHINXBUILD) -Dlanguage=$* -b man $(ALLSPHINXOPTS) $(BUILDDIR)/man/$*
-	@echo
-	@echo "Build finished. The manual pages are in $(BUILDDIR)/man/$*."
-
-texinfo: $(foreach lang,$(LANGS),texinfo-$(lang))
-	$(SPHINXBUILD) -b texinfo $(ALLSPHINXOPTS) $(BUILDDIR)/texinfo/en
-	@echo
-	@echo "Build finished. The Texinfo files are in $(BUILDDIR)/texinfo/en."
-	@echo "Run \`make' in that directory to run these through makeinfo" \
-	      "(use \`make info' here to do that automatically)."
-texinfo-%: locale-%
-	$(SPHINXBUILD) -Dlanguage=$* -b texinfo $(ALLSPHINXOPTS) $(BUILDDIR)/texinfo/$*
-	@echo
-	@echo "Build finished. The Texinfo files are in $(BUILDDIR)/texinfo/$*."
-	@echo "Run \`make' in that directory to run these through makeinfo" \
-	      "(use \`make info' here to do that automatically)."
-
-info: $(foreach lang,$(LANGS),info-$(lang))
-	$(SPHINXBUILD) -b texinfo $(ALLSPHINXOPTS) $(BUILDDIR)/texinfo/en
-	@echo "Running Texinfo files through makeinfo..."
-	make -C $(BUILDDIR)/texinfo/en info
-	@echo "makeinfo finished; the Info files are in $(BUILDDIR)/texinfo/en."
-info-%: locale-%
-	$(SPHINXBUILD) -Dlanguage=$* -b texinfo $(ALLSPHINXOPTS) $(BUILDDIR)/texinfo/$*
-	@echo "Running Texinfo files through makeinfo..."
-	make -C $(BUILDDIR)/texinfo/$* info
-	@echo "makeinfo finished; the Info files are in $(BUILDDIR)/texinfo/$*."
-
-gettext:
-	$(SPHINXBUILD) -b gettext $(I18NSPHINXOPTS) $(PODIR)/
-	@sed -i '/^# [0-9a-f]\{32\}/d' $(PODIR)/ubuntu-packaging-guide.pot
-	@echo "# Will be replaced with a list of translators" >> $(PODIR)/ubuntu-packaging-guide.pot
-	@echo "msgid \"translator-credits\"" >> $(PODIR)/ubuntu-packaging-guide.pot
-	@echo "msgstr \"\"" >> $(PODIR)/ubuntu-packaging-guide.pot
-	@echo "Build finished. The message catalogs are in $(PODIR)/."
-
-changes:
-	$(SPHINXBUILD) -b changes $(ALLSPHINXOPTS) $(BUILDDIR)/changes
-	@echo
-	@echo "The overview file is in $(BUILDDIR)/changes."
-
-linkcheck:
-	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
-	@echo
-	@echo "Link check complete; look for any errors in the above output " \
-	      "or in $(BUILDDIR)/linkcheck/output.txt."
-
-doctest:
-	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
-	@echo "Testing of doctests in the sources finished, look at the " \
-	      "results in $(BUILDDIR)/doctest/output.txt."
-
-locale: $(foreach lang,$(LANGS),locale-$(lang))
-locale-%:
-	@if [ -f po/$*.po ]; then \
-		echo "Compiling .mo files in $(PODIR)/$*/LC_MESSAGES"; \
-		mkdir -p $(PODIR)/$*/LC_MESSAGES; \
-		msgfmt po/$*.po -o po/$*/LC_MESSAGES/ubuntu-packaging-guide.mo; \
-	else echo "$*.po not available, skipping .mo building"; \
-	fi
-
-translators.html:
-	debian/scripts/build-list-of-translators -a -o translators.html
+%:
+  make -C docs $@
```

Issue: I had problems running this make file (`Makefile:2: *** missing separator.  Stop.`) I could fix this by replacing the spaces with a tab.
```suggestion
	make -C docs $@
```